### PR TITLE
Make Admin page

### LINKED ISSUE #2051: Move news out of trello

### LINKED ISSUE #5405: Remove Trello Dependency
Stop using Trello to track News Items (at the top of the chat page).
Implement an admin page to create, edit, and delete news items.
Store the items in mongo.

### PR DESCRIPTION
**Pull request by jwarwick**
Moved **Site News** management into a new **Admin** page and out of Trello.

Fixes #5405 
Fixes #2051

### CODE REVIEW

File: src/clj/web/data.clj [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
             [web.utils :refer [response]]
             [monger.collection :as mc]
             [monger.result :refer [acknowledged?]]
+            [monger.query :refer :all]
             [web.config :refer [server-config]]
             [clojure.edn :as edn]))
 
+(defn news-handler [req]
+  (let [data (with-collection db "news"
+                  (find {})
```
Comment (NoahTheDuke) [inline src/clj/web/data.clj:13]: Spacing?

File: src/clj/web/data.clj [hunk 2]
```diff
@@ -1,11 +1,20 @@
 (ns web.data
+  (:refer-clojure :exclude [sort find])
   (:require [web.db :refer [db object-id]]
             [web.utils :refer [response]]
             [monger.collection :as mc]
             [monger.result :refer [acknowledged?]]
+            [monger.query :refer :all]
```
Comment (NoahTheDuke) [inline src/clj/web/data.clj:7]: Would it be better to not exclude functions from Clojure core but use a prefix?l here? What do you think?

File: src/cljs/nr/auth.cljs [hunk 3]
```diff
@@ -43,7 +43,7 @@
      [:b.caret]]
     [:div.dropdown-menu.blue-shade.float-right
      (when (:isadmin user)
-       [:a.block-link "[Admin]"])
+       [:a.block-link {:href "/admin"} "[Admin]"])
```
Comment (NoahTheDuke) [inline src/cljs/nr/auth.cljs:46]: I've been meaning to make this change for years. Bless you for this.

File: src/clj/web/data.clj (hunk 1 above)
Comment (jwarwick) [reply src/clj/web/data.clj]: Fixed.

File: src/clj/web/data.clj (hunk 2 above)
Comment (jwarwick) [reply src/clj/web/data.clj]: Agreed. Just copying from examples on the web.

**Pull request merged**